install-dev/upgrade/sql/1.7.1.2.sql

```diff
@@ -0,0 +1,3 @@
+SET NAMES 'utf8';
+
+ALTER TABLE `PREFIX_customization_field` ADD `is_deleted` TINYINT(1) NOT NULL DEFAULT '0';
```

Comment:
yes i know but it was moved on develop, it's for this reason, it needs to be rebased, adminproductwrapper is in conflict :/